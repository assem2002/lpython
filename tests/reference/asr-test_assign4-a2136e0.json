{
    "basename": "asr-test_assign4-a2136e0",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_assign4.py",
    "infile_hash": "9e1e5ce063f88194008db998896cebd0fabe6dcb4cbda27c17de27da",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_assign4-a2136e0.stderr",
    "stderr_hash": "396ec086f73ff4e58f5a4817d4d25af3f17ba6f8daaed687ddba51ca",
    "returncode": 2
}