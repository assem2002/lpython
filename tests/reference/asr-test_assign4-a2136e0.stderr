semantic error: Assigning integer to float is not supported
 --> tests/errors/test_assign4.py:7:9
  |
7 |     f = x
  |         ^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
